src/frontend/komponenter/Fagsak/Personlinje/Behandlingsmeny/HenleggBehandling/HenleggBehandling.tsx

```diff
@@ -105,157 +92,168 @@ const HenleggBehandling: React.FC<IProps> = ({ fagsakId, behandling }) => {
-                            ? 'Behandlingen er henlagt og brev til bruker er sendt'
-                            : 'Behandlingen er henlagt'}
-                    </StyledBodyShort>
-                    <Knapperad>
+            {visVeivalgModal && (
+                <Modal
+                    open={visVeivalgModal}
+                    width={'35rem'}
+                    header={{
+                        heading: 'Behandling henlagt',
+                        size: 'medium',
+                    }}
+                    onClose={() => {
+                        settVisVeivalgModal(false);
+                    }}
+                    portal
+                >
+                    <Modal.Body>
+                        <StyledBodyShort>
+                            {årsak === HenleggÅrsak.SØKNAD_TRUKKET
+                                ? 'Behandlingen er henlagt og brev til bruker er sendt'
+                                : 'Behandlingen er henlagt'}
+                        </StyledBodyShort>
+                    </Modal.Body>
+                    <Modal.Footer>
                         <Button
-                            key={'Gå til oppgavebenken'}
+                            key={'Gå til saksoversikten'}
                             variant="secondary"
                             size="medium"
                             onClick={() => {
-                                navigate('/oppgaver');
+                                navigate(`/fagsak/${fagsakId}/saksoversikt`);
                             }}
-                            children={'Se oppgavebenk'}
+                            children={'Se saksoversikt'}
                         />
-                        <KnappHøyre
-                            key={'Gå til saksoversikten'}
+                        <Button
+                            key={'Gå til oppgavebenken'}
                             variant="secondary"
                             size="medium"
                             onClick={() => {
-                                navigate(`/fagsak/${fagsakId}/saksoversikt`);
+                                navigate('/oppgaver');
                             }}
-                            children={'Se saksoversikt'}
+                            children={'Se oppgavebenk'}
                         />
-                    </Knapperad>
-                </Modal.Content>
-            </StyledModal>
-            <PdfVisningModal
-                åpen={visDokumentModal}
-                onRequestClose={() => settVisDokumentModal(false)}
-                pdfdata={hentetDokument}
-            />
+                    </Modal.Footer>
+                </Modal>
+            )}
+            {visDokumentModal && (
+                <PdfVisningModal
+                    åpen={visDokumentModal}
+                    onRequestClose={() => settVisDokumentModal(false)}
+                    pdfdata={hentetDokument}
+                />
+            )}
```

Comment:
Har med vilje latt det være hvitt på toppen for å sørge for at x-knappen er lett tilgjengelig

Før
![image](https://github.com/navikt/familie-ba-sak-frontend/assets/25459913/f07b9e6d-59e5-466c-bef4-785f790d4c2c)

Etter
![image](https://github.com/navikt/familie-ba-sak-frontend/assets/25459913/b42277a6-24b0-44e1-a1e5-c1e09d8af0f5)